FanartHandler/FanartSelectedOther.cs

```diff
@@ -880,6 +929,9 @@ public void ClearSelectedMoviePropertys()
       Utils.SetProperty("movie.clearlogo.selected", string.Empty);
       Utils.SetProperty("movie.banner.selected", string.Empty);
       Utils.SetProperty("movie.cd.selected", string.Empty);
+
+      Utils.SetProperty("series.clearart.selected", string.Empty);
+      Utils.SetProperty("series.clearlogo.selected", string.Empty);

```

Comment:
Everything is logical here, it’s strange that I didn’t add this a long time ago. But then I had the task of bringing FH to life so that he began to work adequately. Maintained compatibility with older versions and added new functionality. Apparently I missed it.